internal/pool/defaults.go

```diff
@@ -1,6 +1,22 @@
 package pool
 
-const DefaultLimit = 50
+import "time"
+
+const (
+	DefaultLimit = 50
+
+	// defaultCreateRetryDelay specifies amount of time that spawner will wait
+	// before retrying unsuccessful item create.
+	defaultCreateRetryDelay = 500 * time.Millisecond
```

Comment:
Can this setting helps for exactly once case?
I think this is over-engineering without the done cases

Reply:
I guess it may be ok to retry instantly (createItem will wait for create timeout anyway).